libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -43,6 +43,8 @@ extern const AP_HAL::HAL& hal;
 # endif
 #endif
 
+#define MIN_MB_FREE_DEFAULT 500
```

Comment:
Just put this inline if it is only used in once place.

Reply:
fixed